server/index.js

```diff
@@ -15,6 +15,11 @@ process.on('unhandledRejection', (error, promise) => {
   logger.error(error);
 });
 
+process.on('uncaughtException', (error, promise) => {
```

Comment:
Why did you add that again? We already have it ! 

![Screenshot 2020-11-02 at 10 22 39](https://user-images.githubusercontent.com/7365207/97845562-63b05d80-1cf5-11eb-89e7-53a3dd91bb1b.png)